goalc/compiler/Compiler.h

```diff
@@ -88,6 +90,7 @@ class Compiler {
   SymbolInfoMap m_symbol_info;
   std::unique_ptr<ReplWrapper> m_repl;
   MakeSystem m_make;
+  GameTextDB m_text_db;
```

Comment:
do we need to keep this here? Or could we construct a temporary one of these inside of `TextTool::run`?

Reply:
or even in `compile_game_text`?

Reply:
The idea is that you could run for example multiple `asm-text-file` like this

```lisp
(asm-text-file text 10 "text_base.txt")
(asm-text-file text 10 "text_extra1.txt")
(asm-text-file text 10 "text_extra2.txt")
```

so the database would need to be saved somewhere like in the compiler. Right now the build system only uses 1 command to build all text so you can probably run it inside the tool for the same effect, but idk.

Reply:
I think it's possible to change it though if the "text project" system is reworked a bit.

Reply:
I think it would be better to change it for a few reasons:
- it makes it easier to track dependencies
- it reduces the amount of state that the compiler has
- it moves closer to a "run the compiler once per output file" model, which I think is what we want to do for jak 2.

I think it's better to end up reading the `text_base.txt` file multiple times.

Reply:
OK I changed it. Also `asm-text-file` already worked a bit differently anyway, you pass it a list of inputs and not just a single input, like `:files ("text_base.txt" "text_extra1.txt")` so I agree it makes more sense to get rid of this stuff in the compiler structure.